backend/package.json

```diff
@@ -15,6 +15,14 @@
     "sourceMap": false,
     "instrument": false
   },
+  "eslintConfig": {
+    "env": {
+      "browser": true,
```

Comment:
erm.... I have no idea how ESlint works but this looks odd to me

Reply:
[Specifying Environments](https://eslint.org/docs/user-guide/configuring#specifying-environments)

"An environment defines global variables that are predefined."

`"broswer": true,` should be removed, as browser globals are not available in Node

the other config options are fine, though.

Reply:
oops, meant to remove that. I was pulling in stuff from the config page and forgot.